lua/entities/gmod_wire_expression2/core/table.lua

```diff
@@ -1122,7 +1122,6 @@ registerCallback( "postinit", function()
 			rv1.n[n] = rv2
 			rv1.ntypes[n] = id
 			self.GlobalScope.vclk[rv1] = true
-			return rv2
```

Comment:
Actually, I believe the mistake here was in not setting the return type. The equivalent array functions do in fact return a value here, and they also have the proper return type set. So, instead of deleting this return, I suggest adding it back in and then giving the function the correct return type. Same for the two functions below.